### PR TITLE
Add Play/Pause button toggle

### PR DESCRIPTION
**Pull request by nikolas**

### CODE REVIEW

File: media/js/src/assetDetail/AssetDetail.jsx [hunk 1]
```diff
@@ -190,8 +190,12 @@ export default class AssetDetail extends React.Component {
...
     }
 
-    onPlayerPlay() {
-        this.setState({playing: true});
+    onPlayToggle() {
+        let newState = true;
+        if (this.state.playing) {
+            newState = false;
+        }
+        this.setState({playing: newState});
```
Comment (sdreher) [inline media/js/src/assetDetail/AssetDetail.jsx:198]: Maybe shorten this to:  
`this.setState({playing: this.state.playing});`

Comment (nikolas) [reply media/js/src/assetDetail/AssetDetail.jsx:198]: Oh yeah - I assume you mean: `this.setState({playing: !this.state.playing});`

Good idea.

Comment (sdreher) [reply media/js/src/assetDetail/AssetDetail.jsx:198]: Der, yep. :-)
